### DISCUSSION TITLE
Hash External Identifier Option - SSO Logins

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#21795] [Feature Requests]
### Summary

In order to have a lower risk, is it possible to hash the external identifier when users log into their account with an SSO like google?
I would prefer not to have their E-mail adress at all.

### Basic Example

rick.roll@gmail.com --> se8f67ts78f

### Motivation

In order to comply with privacy policies and to reduce risks at potentials breaches not storing sensitive information would be nice.

### Detailed Design

Either via a hook or a parameter in the docker-compose file

### Must Have:
- Hook support to implement this feature on my own

### Could Have:
- An automatic integration in directus

### Drawbacks

extra maintenance cost

### Alternatives

custom hook if possible?

### Adoption Strategy

no breaking change in my opinon

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
